scripts/fetch-language-data.sh

```diff
@@ -2,8 +2,9 @@
 
 set -o errexit -o nounset -o pipefail
 
-BASEDIR=$(dirname "$(dirname "$(readlink -f "$0")")")
-DEST="$BASEDIR/src/"
+readlinkf() { perl -MCwd -e 'print Cwd::abs_path shift;' "$1"; }
```

Comment:
Can we use a better readable name instead of readlinkf? For a reader of this script in future, readlink's non-posix issue does not matter. What we are doing here, getting the directory of this script. In next line we navigate to parent directory. `scriptdir()` may be a candidate.

Reply:
Done.